src/main/java/com/blackducksoftware/integration/jira/issue/conversion/BomNotificationToIssueModelConverter.java

```diff
@@ -147,7 +148,7 @@ public BomNotificationToIssueModelConverter(final JiraServices jiraServices, fin
                 final Collection<BlackDuckIssueModel> createdIssueWrappers = populateModelFromContentDetail(jiraProject, projectVersionWrapper, notificationType, detail, notificationContent, batchStartDate);
                 issueWrapperList.addAll(createdIssueWrappers);
             } catch (final Exception e) {
-                logger.error(e);
+                logger.error("An error occurred while converting from notifications to models", e);
```

Comment:
Can we change this to something like "An error occurred while converting from notification contents to models". This way when we see the message in the log it will be easier to track down where it came from.